lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -44,12 +44,12 @@ public function init() {
         'id' => 'wpa0_default_login_redirection', 'function' => 'render_default_login_redirection' ),
       array( 'name' => __( 'Passwordless Login', 'wp-auth0' ), 'opt' => 'passwordless_enabled',
         'id' => 'wpa0_passwordless_enabled', 'function' => 'render_passwordless_enabled' ),
+      array( 'name' => __( 'Connections to Show', 'wp-auth0' ), 'opt' => 'lock_connections',
+        'id' => 'wpa0_connections', 'function' => 'render_connections' ),
       array( 'name' => __( 'Force HTTPS Callback', 'wp-auth0' ), 'opt' => 'force_https_callback',
         'id' => 'wpa0_force_https_callback', 'function' => 'render_force_https_callback' ),
-      array( 'name' => __( 'Lock JS CDN URL', 'wp-auth0' ), 'opt' => 'passwordless_enabled',
+      array( 'name' => __( 'Lock JS CDN URL', 'wp-auth0' ), 'opt' => 'cdn_url',
```

Comment:
can these 2 var renaming changes be considered breaking changes?

Reply:
They were incorrect in a previous (unreleased) branch. More testing found the incorrect option name.